aws/networking/main.tf

```diff
@@ -0,0 +1,79 @@
+data "aws_availability_zones" "available" {
+  state = "available"
+}
+
+resource "aws_vpc" "main" {
+  cidr_block           = "10.0.0.0/16"
+  enable_dns_hostnames = true
+  enable_dns_support   = true
+  tags = local.tags
+}
+
+resource "aws_internet_gateway" "main" {
+  vpc_id = aws_vpc.main.id
+  tags = local.tags
+}
+
+resource "aws_egress_only_internet_gateway" "main" {
+  vpc_id = aws_vpc.main.id
+  tags = local.tags
+}
+
+resource "aws_route_table" "main" {
+  vpc_id = aws_vpc.main.id
+
+  route {
+    cidr_block = "0.0.0.0/0"
+    gateway_id = aws_internet_gateway.main.id
```

Comment:
Do we want both ingress and egress here? Should we limit this to egress only?

Reply:
Yes, otherwise I think it will not route requests to the internet.

Reply:
My question came from seeing that we only do ingress for IPV6.